runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -220,6 +220,15 @@ typedef struct nnfw_train_info
   NNFW_TRAIN_OPTIMIZER opt = NNFW_TRAIN_OPTIMIZER_SGD;
 } nnfw_train_info;
 
+/**
+ * @brief Set train_info into session
+ * @note  This function should be called in MODEL_LOADED state,
```

Comment:
User cannot see `MODEL_LOADED` state, how about removing the content related to `MODEL_LOADED`?

Reply:
Updated :)